src/operators/fully-connected-nc.c

```diff
@@ -460,6 +460,310 @@ enum xnn_status xnn_create_fully_connected_nc_qd8_f16_qc4w(
     fully_connected_op_out);
 }
 
+enum xnn_status xnn_create_fully_connected_nc_qd8_f16_qb4w(
```

Comment:
do you think we can share logic with other `create` functions and use `create_fully_connected_nc`? For both fp16 and fp32?